affinity/cpp-20/d0796r3.md

```diff
@@ -221,7 +276,7 @@ An `execution_resource` is a lightweight structure which acts as an identifier t
 
 ### System topology
 
-The system topology is made up of a number of system-level `execution_resource`s, which can be queried through `this_system::get_resources` which returns a `std::vector`. A run-time library may initialize the `execution_resource`s available within the system dynamically. However, this must be done before `main` is called, given that after that point, the system topology may not change.
+The system topology is made up of a number of system-level `execution_resource`s, which can be queried through `this_system::get_resources` which returns a `std::vector`. A run-time library may initialize the `execution_resource`s available within the system dynamically. However, `this_system::get_resources` must be thread safe and must initialize and finalize any third-party or OS state before returning.
```

Comment:
If we don't want iterators to be invalidated, shouldn't this be std::list ?

Reply:
Issue #50 and PR #73 address making resources iterable.  I wanted this PR to be as orthogonal from those as possible, though I think I did manage to conflict with #73, alas ;-) .  Anyway, thanks for pointing this out!